README.md

```diff
@@ -82,6 +82,31 @@ You can now navigate to the site (in a browser of your choice) with the containe
 
     185.5.3.12:8777
 
+## LXD Documentation
+
+To generate LXD Documentation html, checkout the LXD Repository from https://github.com/lxc/lxd and run `make doc`: 
+
+    git checkout https://github.com/lxc/lxd
+    cd lxd
+    make doc
+
+The command will emit to the ./doc/html/ folder. Its contents can be copied into the output folder created by `./generate`:
+
+    mkdir -p output/lxd/docs/master
+    cp -R ../lxd/doc/html/* output/lxd/docs/master/
+
+This needs to be repeated after `./generate` runs, because it cleans the output folder.
```

Comment:
```suggestion
## LXD documentation

To generate the HTML of the LXD documentation, check out the LXD repository from https://github.com/lxc/lxd and run `make doc`: 

    git checkout https://github.com/lxc/lxd
    cd lxd
    make doc

The command writes the HTML output to the `./doc/html/` folder. Copy its contents into the output folder created by `./generate`:

    mkdir -p output/lxd/docs/master
    cp -R ../lxd/doc/html/* output/lxd/docs/master/

You must repeat this process after `./generate` runs, because it cleans the output folder.
```